guides/v2.3/graphql/queries/is-email-available.md

```diff
@@ -0,0 +1,54 @@
+---
+group: graphql
+title: isEmailAvailable query
+---
+
+The `isEmailAvailable` query checks whether the specified email has already been used to create a customer account. A value of `true` indicates the email address is available, and the customer can use the email address to create an account.
+
+To return or modify information about a customer, Magento recommends you use customer tokens in the header of your GraphQL calls. However, you also can use [session authentication]({{ page.baseurl }}/get-started/authentication/gs-authentication-session.html).
```

Comment:
I think it's safe to not include this paragraph, because the query is checking whether a specific emall address has been used before.